graviti/dataset/dataset.py

```diff
@@ -9,13 +9,14 @@
 
 from tensorbay.dataset import Notes, RemoteData
 from tensorbay.label import Catalog
+from tensorbay.utility import URL
```

Comment:
your commit message is not clear.
try to directly describe what have you done,
like `implement xxx` or `add xxx`